CHANGELOG.md

```diff
@@ -3,6 +3,58 @@ Changes by Version
 
 <!-- next version -->
 
+## 0.3.0
+
+### 🛑 Breaking changes 🛑
+
+- `operator`: Upgrade tempo to v2.1.1 (#408)
+  The `maxSearchBytesPerTrace` global and per-tenant limit is deprecated.
+  A new `maxSearchDuration` global and per-tenant limit is available.
+  Some metrics got renamed or deleted, see the [Tempo v2.1.0 release notes](https://github.com/grafana/tempo/releases/tag/v2.1.0) for details.
+  
+
+### 💡 Enhancements 💡
+
+- `operator`: Add .spec.managedState to TempoStack. It allows disabling reconciliation of the TempoStack CR. The validation and defaulting webhooks remain enabled in unmanaged state. (#411)
+- `operator`: Enable mTLS by default for internal components (#505)
+- `operator`: Expose the Jaeger GRPC query port (#513)
+- `operator`: Expose supported protocols on the distributor (#436)
+  The following protocols are now exposed: 
+    - Jaeger: thrift-http port 14268, thrift-binary port 6832, thrift-compact port 6831, Grpc port 14250
+    - Zipkin port 9411
+  
+- `operator`: Use internal certificate for internal HTTP server of gateway (#480)
+- `operator`: Add ability to create and configure route or ingress for the gateway (#265)
+- `operator`: The operator is now at Level 4 - Deep Insights (#504)
+  The operator optionally exposes metrics and alerts for the operator and the operand.
+  
+- `operator`: Enable mTLS on receivers when gateway is enabled (#535)
+- `operator`: Enable multitenancy without need the gateway (#224)
+- `operator`: Add operator alerts and runbook. (#309)
+  The alerts are only installed for the OpenShift bundle.
+  See https://github.com/grafana/tempo-operator/issues/491 for a discussion to enable them in the community bundle.
```

Comment:
```suggestion

```

Reply:
Let's remove these two lines, as another change in the same release allows enabling them for the community bundle with just an operator configuration setting :)